app/models/event.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
```

Comment:
Doesn't this class need a top level comment too?

Reply:
Yeah! It's super weird Rubocop didn't complain.